yolov8_ros/yolov8_ros/yolov8_ros/yolov8_node.py

```diff
@@ -259,16 +299,26 @@ def image_cb(self, msg: Image) -> None:
             # create detection msgs
             detections_msg = DetectionArray()
 
+            iceberg_detections_msg = IcebergDetectionArray()
+
             for i in range(len(results)):
 
                 aux_msg = Detection()
 
+                iceberg_aux_msg = IcebergDetection()
+
                 if results.boxes:
                     aux_msg.class_id = hypothesis[i]["class_id"]
                     aux_msg.class_name = hypothesis[i]["class_name"]
                     aux_msg.score = hypothesis[i]["score"]
 
                     aux_msg.bbox = boxes[i]
```

Comment:
I think this can be removed too?

Reply:
Keeping old publisher for visualization purposes. Reverted to commit 744ad748b19208b1077d615650d5b2944bdaa700.